test/e2e-common.sh

```diff
@@ -25,10 +25,9 @@ E2E_CLUSTER_MACHINE=${E2E_CLUSTER_MACHINE:-n1-standard-8}
 # This script provides helper methods to perform cluster actions.
 source $(dirname $0)/../vendor/knative.dev/test-infra/scripts/e2e-tests.sh
 
-# Default Istio configuration to install: 1.2-latest, no mesh, cert manager 0.6.1.
-ISTIO_VERSION="1.2-latest"
-ISTIO_MESH=0
 CERT_MANAGER_VERSION="0.9.1"
+ISTIO_VERSION=""
```

Comment:
This is now a breaking change: by default no Istio or Gloo will be installed (before, Istio was the default). Is it intended?

Reply:
Yes. I thought that in your latest change, we now always pass flags to the test scripts.
This one: https://github.com/knative/test-infra/pull/1481/files

Reply:
Oops, that fell off my radar. I need to get that merged first then.

Reply:
1481 is now merged and jobs are updated.